### ISSUE TITLE
Memory leak in _blinkTiming

### ISSUE THREAD

**Issue by vfschenkelaars** [mickey9801/BlinkControl#5]
creation of _blinkTiming leaks memory is object is set from blink to on to blink. The destruction is also unchecked so calling clear after only calling on will try to free unallocated memory. Better to use a fixed array for the blink pattern.

**Comment by paoSoft** [comment]
Agree! The blink() method should just take the pattern-array as a reference, and use it wherever it is located. No new. No copy. The convenient class patterns should be static const, and any "dynamic" user provided patterns, is left to the user to memory manage.
